presto-function-namespace-managers/src/main/java/com/facebook/presto/functionNamespace/json/FileBasedJsonInMemoryFunctionNameSpaceManagerFactory.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.functionNamespace.json;
+
+import com.facebook.airlift.bootstrap.Bootstrap;
+import com.facebook.presto.functionNamespace.execution.NoOpSqlFunctionExecutorsModule;
+import com.facebook.presto.spi.function.FunctionHandleResolver;
+import com.facebook.presto.spi.function.FunctionNamespaceManager;
+import com.facebook.presto.spi.function.FunctionNamespaceManagerContext;
+import com.facebook.presto.spi.function.FunctionNamespaceManagerFactory;
+import com.facebook.presto.spi.function.SqlFunctionHandle;
+import com.google.inject.Injector;
+
+import java.util.Map;
+
+import static com.google.common.base.Throwables.throwIfUnchecked;
+
+public class FileBasedJsonInMemoryFunctionNameSpaceManagerFactory
```

Comment:
Can you add some java doc describing what does this do?

Reply:
What do you mean by java doc? Do you mean the presto documentation?

Reply:
Added java documentation comments.